test/framework/convenience.go

```diff
@@ -17,27 +17,28 @@ limitations under the License.
 package framework
 
 import (
+	"context"
 	"fmt"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 // InstallComponents is a helper function that applies components, generally to a management cluster.
-func InstallComponents(mgmt Applier, components ...ComponentGenerator) {
+func InstallComponents(ctx context.Context, mgmt Applier, components ...ComponentGenerator) {
 	Describe("Installing the provider components", func() {
 		for _, component := range components {
 			By(fmt.Sprintf("installing %s", component.GetName()))
```

Comment:
I suggest removing the `Describe` and changing the following
```suggestion
			fmt.Printf("Installing %s"\n, component.GetName())
```

This will make it easier to read the output. Also we can get rid of the `ginkgo` import as well.

Reply:
Actually, if we do that, then this wouldn't give us any output when run in ginkgo by default, which may make troubleshooting the jobs a bit more difficult.

We could potentially extend the framework to take an io.Writer that defaults to `os.Stdout`, and downstream consumers using ginkgo could pass in `GinkgoWriter`.

Reply:
This is already possible by setting `ginkgo.GinkgoWriter = os.Stdout` or any other `io.Writer`.

I don't think this particular change makes sense for the reasons @detiber has outlined.

@wfernandes Is there a particular issue you'd like to address here? I didn't find the output hard to read, but if you can show me why you think it's hard to read that would help a lot, thank you!

Reply:
@chuckha I pulled your latest changes and I think the flag `-ginkgo.v` did the trick and outputted the "installing %s" line which makes the output more readable. 
This is not an issue now 😄

Reply:
Just to clarify, I was using the `make -C test/infrastructure/docker test-e2e` command to run the tests and not `ginkgo`.

Reply:
great!